test/unit/math.spec.ts

```diff
@@ -8,7 +8,13 @@ export class MathTests {
     @TestCase("Math.cos()", "math.cos();")
```

Comment:
Missing test for constants (probably would be nice to actually `util.transpileAndExecute` those)

Reply:
There are tests for `Math.PI` and `Math.E`. Should I add a test for every one?

Also, I'm not sure we should do execute tests since there could be inconsistencies with precision.